File: 백준/Bronze/2439. 별 찍기 － 2/별 찍기 － 2.java
import java.util.Scanner;
class Main{
    public static void main(String args[]){
       Scanner sc = new Scanner(System.in);
    
        int star = sc.nextInt();
    
        for(int i=1; i<=star; i++){
            for(int j=star-1; j>=i; j--)
                System.out.print(" ");
            for(int k=1; k<=i; k++)
                System.out.print("*");
            System.out.println();
        }
    }
}

File: 백준/Bronze/2439. 별 찍기 － 2/별 찍기 － 2.cc
#include <stdio.h>
main(){
	int star;
	scanf("%d", &star);
	
	for(int i=1; i<=star; i++){
        for(int j=star-1; j>=i; j--)
            printf(" ");
        for(int k=1; k<=i; k++)
            printf("*");
        printf("\n");
    }
}